### PR TITLE
README.rst: Added screenshot of the demo page

### PR DESCRIPTION
**Pull request by marian-ald**

### CODE REVIEW

**Comment by tulphoon** [comment]
Hello @marian-ald.

I think that is not what was meant by screenshot of the demo page. The page is located at http://coala.github.io/coala-html/#/

You should also include a `Fixes/Closes` message in your commit message. I would also change from 'Added' to 'Add' in your commit shortlog.

**Comment by marian-ald** [comment]
So, have I to do it for the link above? http://coala.github.io/coala-html/#/

**Comment by marian-ald** [comment]
Unfortunately I don't know what do you mean by Fixes / Closes

**Comment by aptrishu** [comment]
Go to https://coala.io/commit . @marian-ald

**Comment by Nosferatul** [comment]
Now the screenshot it's  fine, squash your commits and add `Closes <link_to_your_issue>` in the commit  message. use `git commit --amend` change your commit message and push again!

**Comment by tulphoon** [comment]
@marian-ald okay, you still need to squash your commits. to do that, run `git rebase -i HEAD~3`, that should try to rebase latest three commits from your branch. after you do it, you should see, something like this
'''
pick u59gw6 Added demo of...
pick 725529 Add demo of...
pick 9hoeh7 Add demo of...

<bunch of comments on usage here>
```

In the usage comments you can see an 'squash' option.

Replace two of these 'pick' with 'squash' and save the file. Then you will end up editing another file. This is the commit message one. I would make it like this
```
README.rst: Add screenshot of the demo page.

Closes <link to issue here>
```

Then save and force push.

File: README.rst [hunk 1]
```diff
@@ -44,6 +44,10 @@ DEMO
 ----
 GIF_.  Live_.
```
Comment (Adrianzatreanu) [inline README.rst]: hey it seems that somehow when you were rebasing you got some changes which werent yours.. could you fix this, or if you dont know how, ask on gitteR :)? Thanks!

**Comment by Mixih** [comment]
@marian-ald to remove the files that aren't yours in this pull see:  http://stackoverflow.com/questions/18357511/git-remove-commited-file-after-push

**Comment by marian-ald** [comment]
I followed those steps. I think it should be ok, but still have those commits on my page :/...

**Comment by aptrishu** [comment]
Ask us on the channel. https://gitter.im/coala/coala someone will help you.

**Comment by jayvdb** [comment]
Unfortunately, you did not follow the [Git basics](http://api.coala.io/en/latest/Developers/Git_Basics.html).  This Pull Request (PR) is using your "master" branch, which means you did not [create a new branch](http://api.coala.io/en/latest/Developers/Git_Basics.html#creating-a-new-branch).

Please redo your changes, following the Git Basics guide, and submit a new PR.

**Pull request closed without merging**